.github/workflows/main.yml

```diff
@@ -124,6 +124,14 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
 
+      - name: Set up Google Chrome
+        uses: browser-actions/setup-chrome@v1
+        if: contains(matrix.browser, 'Chrome')
+
+      - name: Set up Firefox
+        uses: browser-actions/setup-firefox@v1
+        if: contains(matrix.browser, 'Firefox')
+
```

Comment:
Added some logic here to ensure we always get the latest version of Chrome and Firefox. Seems like we can get different versions of these browsers depending on what runner GitHub actions graces us with otherwise, which leads to inconsistent results.